public/sass/base/_base.scss

```diff
@@ -10,7 +10,59 @@ html {
     box-sizing: border-box;
     font-size: 62.5%;
 }
+body{
+    background-color: yellow;
+}
+
+.splash{
```

Comment:
move the css related to splash screen in the layouts folder in a new scss file named _splash.scss